source/val/validate_decorations.cpp

```diff
@@ -331,6 +336,8 @@ uint32_t getSize(uint32_t member_id, const LayoutConstraints& inherited,
       const auto& constraint = constraints[std::make_pair(lastMember, lastIdx)];
       return offset + getSize(lastMember, constraint, constraints, vstate);
     }
+    case SpvOpTypePointer:
+      return 8;
```

Comment:
Ideally this would be a bit more abstracted, e.g. something set up early in the validation state.  A Physical32 addressing mode would have this as 4, but those environments doesn't run through this code.

Reply:
I've added this.